pegasus/sites.v3/hourofcode.com/public/events/organizations.haml

```diff
@@ -5,17 +5,16 @@ nav: events_nav
 ---
 -SOLR = Solr::Server.new(host:CDO.solr_server)
 
--by_company = {}
+-company_event_counts = Hash.new {|h,k| h[k] = 0}
 
 -events = SOLR.query(q:"*:*", fq:["kind_s:HocSignup2016", "location_country_s:[* TO *]", "organization_name_s:[* TO *]", "hoc_company_s:[* TO *]"], sort:'location_country_s asc, location_state_s asc', rows:1000)
 -events.each do |event|
   -company = event['hoc_company_s'].to_s
-  -by_company[company] = [] unless by_company.has_key?(company)
-  -by_company[company] << nil
+  -company_event_counts[company] += 1
 
 %h1= hoc_s(:events_orgs_title)
 
 %ul
-  -by_company.each_pair do |company, events|
+  -company_event_counts.each_pair do |company, event_count|
     %li{class: "hoc-event-country"}
-      =events.count > 1 ? "#{hoc_s('company_'+company)} (#{events.count} #{hoc_s(:events)})" : "#{hoc_s('company_'+company)} (#{events.count} #{hoc_s(:event)})"
+      =events.count > 1 ? "#{hoc_s('company_'+company)} (#{event_count} #{hoc_s(:events)})" : "#{hoc_s('company_'+company)} (#{event_count} #{hoc_s(:event)})"
```

Comment:
since `events` used to shadow the outer variable but no longer does, should `events.count > 1` also be updated to `event_count > 1`?